content/en/remote-network-agent/install-helm.md

```diff
@@ -0,0 +1,251 @@
+
+- Secrets management outside of Kubernetes (AWS Secrets Manager, AWS S3, Vault, Kubernetes Injector)
+- Agent proxy settings
+- Pod labels
+- Pod annotations
+- Pod environment variables
+- Pod resource requests and limits
+- Pod DNS settings 
+- Pod Node selection 
+- Pod affinity 
+- Pod tolerations
+- Log configuration
+- Disable Kubernetes cluster mode
+- Metrics 
+
+## {{% heading "prereq" %}}
+
+* You have installed [Helm](https://helm.sh/docs/intro/quickstart/) v3+.
+* You configure advanced options in a `values` file, so you should be familiar with using that file to customize a chart. For more information, see the Helm docs:
+
+  * [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing)
+  * [Helm Values Files guide](https://helm.sh/docs/chart_template_guide/values_files/) 
+
+* You have created a namespace for your RNA:  `kubectl create ns armory-rna`. Since you pass the namespace to the `helm` command, you can use a different namespace. This guide assumes you have created an `armory-rna` namespace.
+
+Installation using Helm consists of the following steps:
+
+1. Configure your installation in the  `values.yaml` file, which you should download from the [repo](https://github.com/armory-io/remote-network-agent-helm-chart/blob/main/values.yaml).  
+
+   * [Configure required settings](#configure-required-settings)
+   * [Configure optional settings](#configure-optional-settings)
+
+1. [Generate and preview the manifests](#generate-and-preview-the-manifests)
+1. [Install the RNA](#install-the-rna)
+
+<details><summary>Show the values file</summary>
+{{< github repo="armory-io/remote-network-agent-helm-chart" file="/values.yaml" lang="yaml" options="" >}}
+</details><br>
+
+## Configure required settings
+
+>If you don't configure these in the file, you must pass them on the command line using `--set <key>:<value>`.
+
+* `clientId` and `clientSecret`: the Client ID and Client Secret from your Client Credentials; you should encrypt the Client ID and Client Secret as a secret in a supported secrets manager such as [Kubernetes](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/) or [AWS Secrets Manager](https://docs.aws.amazon.com/eks/latest/userguide/manage-secrets.html). 
+
+   For example, to create a Kubernetes secret, execute the following:
+
+   ```bash
+   kubectl --namespace armory-rna create secret generic rna-client-credentials --type=string --from-literal=client-secret=<your-client-secret> --from-literal=client-id=<your-client-id>
+   ```
+
+   Then you would configure `clientId` and `clientSecret` like this:
+
+   ```yaml
+   clientId: encrypted:k8s!n:rna-client-credentials!k:client-id
+   clientSecret: encrypted:k8s!n:rna-client-credentials!k:client-secret
+   ```
+
+   See the comments in the `values.yaml` file for examples using other secrets managers.
+
+* `agentIdentifier`: the name of your Remote Network Agent
```

Comment:
This name will be used within CD-as-a-Service to identify your cluster when defining deployment targets `targets.<targetname>.account`